.github/workflows/publish-operator-for-e2e-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: publish-operator-for-e2e-tests
+on:
+  pull_request_target
+
+env:
+  GOPATH: /tmp/go
+  GO_VERSION: 1.15.x
+
+jobs:
+  binary:
+    name: Build & push operator bundle for e2e tests
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
+
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles ('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Prepare tools
+      #      uses: codeready-toolchain/toolchain-cicd/prepare-tools-action@master
+      uses: MatousJobanek/toolchain-cicd/prepare-tools-action@master
```

Comment:
are you going to revert that?

Reply:
not now. I need to:
1. make it working 
2. create & merge PR for toolchain-cicd with all changes 
3. and then we can switch to `codeready-toolchain/toolchain-cicd`

Reply:
ok, thanks for clarifying ;)